lib/Util/SuggestionNameFormatter.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerCompletion\Util;
+
+use Phpactor\Completion\Core\Suggestion;
+
+class SuggestionNameFormatter
+{
+    public function format(Suggestion $suggestion): string
+    {
+        $name = $suggestion->name();
+
+        switch ($suggestion->type()) {
+            case Suggestion::TYPE_VARIABLE:
+                $name = mb_substr($name, 1);
+                break;
+            case Suggestion::TYPE_FUNCTION:
+            case Suggestion::TYPE_METHOD:
+                $name = $name . '(';
```

Comment:
I think we should remove this also: `(` is used as to trigger signature help.

Reply:
You are right. It is a hack although personally obtaining `(` is for me more practical than displaying each time signature help. I have defined a mapping to trigger the help:
```vim
" abstraction to reuse later by any client, e.g. phpactpr RPC
imap <buffer> <C-\>, <Plug>(ide-show-signature-help)
" in the case of using CoC
imap <Plug>(ide-show-signature-help) <C-o>:call CocActionAsync('showSignatureHelp')<CR>
```